src/Libraries/CoreNodeModels/DynamoConvert.cs

```diff
@@ -76,8 +171,9 @@ protected override void SerializeCore(XmlElement element, SaveContext context)
 
             var xmlDocument = element.OwnerDocument;
             var subNode = xmlDocument.CreateElement("Range");
```

Comment:
Is there a reason you're creating a subNode element rather than just using XML attributes? This makes migrations a lot more difficult.